frontend/scss/components/molecules/filter-bubble.scss

```diff
@@ -29,7 +29,8 @@ The filter-bubble molecule is a simple button used to filter content based on th
   background: color('white');
   color: color('oslo-gray');
   cursor: pointer;
-  margin: 5px 5px;
+  margin: 5px 2px;
```

Comment:
Beware, this component is used on every overview page. Considering devices with this screen size are usually navigated by touch I'd suggest reverting this change. The problem with the big spacing for the roadmap seems to have to do with significant whitespace. Compare the following screenshots from components overview and your roadmap:

![Bildschirmfoto 2020-05-06 um 22 10 13](https://user-images.githubusercontent.com/12857772/81223537-6fd98d80-8fe6-11ea-8753-4c49c3fe3f39.png)
![Bildschirmfoto 2020-05-06 um 22 10 21](https://user-images.githubusercontent.com/12857772/81223548-7405ab00-8fe6-11ea-8a7f-cab3f249c562.png)